packages/proxy/lib/http/util/cookies.ts

```diff
@@ -10,28 +10,6 @@ interface RequestDetails {
   needsCrossOriginHandling: boolean
 }
 
-/**
- * Whether or not a url's scheme, domain, and top-level domain match to determine whether or not
- * a cookie is considered first-party. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#third-party_cookies
- * for more details.
- * @param {string} url1 - the first url
- * @param {string} url2 - the second url
- * @returns {boolean} whether or not the URL Scheme, Domain, and TLD match. This is called same-site and
- * is allowed to have a different port or subdomain. @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-Fetch-Site#directives
- * for more details.
- */
-export const areUrlsSameSite = (url1: string, url2: string) => {
```

Comment:
What's the reasoning to moving this to `cors`? Is it only used for cookies? If so, it seems better placed here.